tests/behat/features/web/general/search.feature

```diff
@@ -48,17 +103,27 @@ Feature: Searching for programs
     Given I am on "/app"
     And I wait for the page to be loaded
     Then I click ".search-icon-header"
-    And I enter "prog" into visible ".input-search"
+    And I enter "program" into visible ".input-search"
     And I click "#btn-search-header"
     And I wait for the page to be loaded
-    Then I should be on "/app/search/prog"
-    And I should see "Your search returned 4 results"
+    Then I should be on "/app/search/program"
+    And I should see "Your search returned 3 results"
     And at least one ".search-input-header" element should be visible
     And at least one "#btn-search-header" element should be visible
-    Then I enter "yahoo myprog" into visible ".input-search"
+    Then I enter "Test advanced" into visible ".input-search"
     And I click "#btn-search-header"
     And I wait for the page to be loaded
-    Then I should be on "/app/search/yahoo%20myprog"
+    Then I should be on "/app/search/Test%20advanced"
     And I should see "Your search returned 2 results"
     And at least one ".search-input-header" element should be visible
     And at least one "#btn-search-header" element should be visible
+
+  Scenario: search for gmail should search for all
```

Comment:
scenario name is still the old one. Change it to, "should not find anything" or something like that